src/lib/viewers/BaseViewer.js

```diff
@@ -595,6 +595,8 @@ class BaseViewer extends EventEmitter {
             this.annotator.emit(ANNOTATOR_EVENT.setVisibility, true);
             if (this.options.enableAnnotationsDiscoverability) {
                 this.annotator.toggleAnnotationMode(AnnotationMode.REGION);
+                this.annotationControlsFSM.reset();
+                this.processAnnotationModeChange(AnnotationMode.NONE);
```

Comment:
If needed, I can move this change out to a separate PR. The bug that @mxiao6 described affects both `handleFullScreenExit` as well as `handleAnnotationControlsEscape` so I thought I would lump this change in with the `handleAnnotationControlsEscape` change.

Reply:
Instead of changing the usage pattern, can we instead introduce a new `AnnotationInput.RESET` that can be passed as an argument into `this.processAnnotationModeChange(this.annotationControlsFSM.transition(AnnotationInput.RESET))`?

Reply:
This would be cleaner - will implement.